webserver/__init__.py

```diff
@@ -47,7 +64,7 @@ def create_app(debug=None, config_path=None):
 
     # Logging
     from webserver.loggers import init_loggers
-    init_loggers(app)
+    init_loggers(app) # TODO: use CustomFlask.init_loggers here instead
```

Comment:
At some point we should add support for setting the log level via the config file.

Reply:
[AB-355](https://tickets.metabrainz.org/browse/AB-355)